tools/lsp/ui/api.slint

```diff
@@ -143,6 +143,8 @@ export global Api {
     in-out property <DropMark> drop-mark;
     in property <component-factory> preview-area; // The actual preview
 
+    in property <bool> resize-to-preferred-size: false; // set to true to resize
```

Comment:
Is there a better way to tell the UI to adapt something?

Like a callback from the backend into the UI?

Reply:
I can't think of a better way.